ouroboros-network/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 
 * Fix diffusion tests
 
+* Update types to accommodate `PeerSharing` data type changes.
```

Comment:
This entry should explain the changes in a little more detail, e.g. how it will affect node including previous versions.